scripts/metadata_comparison/metadata_comparison/lib/operations_digesters.py

```diff
@@ -137,6 +182,28 @@ def machine_type(self) -> AnyStr:
         machine_type_with_zone_prefix = self.metadata().get('runtimeMetadata').get('computeEngine').get('machineType')
         return machine_type_with_zone_prefix.split('/')[-1]
 
+    def disks(self) -> dict:
```

Comment:
Can we be more precise about what type of dict we get back?

Reply:
Probably so. Even if I get it wrong I don't think Python would let us know... 😄

Reply:
Yeah it just makes using the function easier in the future if the IDE tells you what you're expected to return 🤷